lib/scrabble.rb

```diff
@@ -1,7 +1,12 @@
 class Scrabble
 
   def score(word)
-    1
+    score = 0
+    letters = word.to_s.split(//)
+    letters.each do |letter|
+      score += point_values[letter.upcase]
+    end
+    score
```

Comment:
Thinking here is very clear. Like how straightforward this all seems.

Could be beneficial to split into even smaller methods to highlight the overarching logic of this method.